ui/entry-index.js

```diff
@@ -68,9 +68,9 @@ import './js/controllers/tcjobactions';
 import './plugins/tabs';
 import './plugins/controller';
 import './plugins/pinboard';
-import './plugins/similar_jobs/controller';
```

Comment:
The `ui/plugins/similar_jobs/controller.js` file can now be removed too :-)

Reply:
Wow, talk about a good call:  This is one.  done.  :)